sagemaker-clarify/fairness_and_explainability/fairness_and_explainability_boto3.ipynb

```diff
@@ -0,0 +1,1541 @@
+    "  * `instance_type` and `initial_instance_count` specify your preferred instance type and instance count used to run your model on during SageMaker Clarify's processing. The testing dataset is small so a single standard instance is good enough to run this example.\n",
+    "  * `accept_type` denotes the endpoint response payload format, and `content_type` denotes the payload format of request to the endpoint. As per the example model we created above both of these will be `text/csv`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:botocore.credentials:Found credentials from IAM Role: BaseNotebookInstanceEc2InstanceRole\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "'s3://sagemaker-ap-south-1-003394947794/sagemaker/DEMO-sagemaker-clarify-boto3/bias_analysis_config.json'"
+      ]
+     },
+     "execution_count": 22,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Upload the analysis_config to the concerned S3 path.\n",
+    "S3Uploader.upload(\"analysis_config/bias_analysis_config.json\", \"s3://{}/{}\".format(bucket, prefix))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Run Sagemaker Clarify Processing Job\n",
+    "Refer this documentation to [configure a clarify procesing job](https://docs.aws.amazon.com/sagemaker/latest/dg/clarify-processing-job-configure-parameters.html) for your use case."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "def create_processing_job(analysis_config_path, analysis_result_path):\n",
+    "    processing_job_name = \"DEMO-clarify-job-{}\".format(datetime.now().strftime(\"%d-%m-%Y-%H-%M-%S\"))\n",
+    "\n",
+    "    response = sagemaker_client.create_processing_job(\n",
+    "        ProcessingJobName=processing_job_name,\n",
+    "        AppSpecification={\"ImageUri\": clarify_image_uri},\n",
+    "        ProcessingInputs=[\n",
+    "            {\n",
+    "                \"InputName\": \"analysis_config\",\n",
+    "                \"S3Input\": {\n",
+    "                    \"S3DataType\": \"S3Prefix\",\n",
+    "                    \"S3InputMode\": \"File\",\n",
+    "                    \"S3Uri\": analysis_config_path,\n",
+    "                    \"LocalPath\": \"/opt/ml/processing/input/config\",\n",
+    "                },\n",
+    "            },\n",
+    "            {\n",
+    "                \"InputName\": \"dataset\",\n",
+    "                \"S3Input\": {\n",
+    "                    \"S3DataType\": \"S3Prefix\",\n",
+    "                    \"S3InputMode\": \"File\",\n",
+    "                    \"S3Uri\": train_uri,\n",
```

Comment:
Regarding the previous question,

> test_uri ?
>> train data is being used for all analysis in existing notebook, hence i followed the same.
>> Bias analysis should be done on train uri only, no?

Then I think the "testing dataset" in these two can be changed to "example dataset" to avoid confusion, 

"..launch SageMaker Clarify jobs to analyze a testing dataset in CSV format"

"...The testing dataset is small so a single standard instance is good enough to run this example."